pkgs/development/web/minify/default.nix

```diff
@@ -46,5 +46,6 @@ buildGoModule rec {
     changelog = "https://github.com/tdewolff/minify/releases/tag/v${version}";
     license = licenses.mit;
     maintainers = with maintainers; [ gaelreyrol ];
+    mainProgram = "minify";
```

Comment:
There is no need to define `mainProgram` as there is only one binary which matches the derivation name.

Reply:
`lib.getExe` deprecated the behavior that assumes that the binary has the same name as the derivation. See #246386

Reply:
Alright, it's sad those kind of things aren't pushed in a "changelog" to help contributors keep up with packaging guidelines.